cldf/cldf-metadata.json

```diff
@@ -0,0 +1,321 @@
+{
+    "@context": "http://www.w3.org/ns/csvw",
+    "aboutUrl": null,
+    "dc:bibliographicCitation": "",
+    "dc:conformsTo": "http://cldf.clld.org/v1.0/terms.rdf#Wordlist",
+    "dc:description": null,
+    "dc:identifier": "",
+    "dc:isVersionOf": null,
+    "dc:license": null,
+    "dc:related": null,
+    "dc:source": "sources.bib",
+    "dc:title": "backstromnorthernpakistan",
+    "dcat:accessURL": "https://github.com/lexibank/backstromnorthernpakistan",
+    "rdf:ID": "backstromnorthernpakistan",
+    "rdf:type": "http://www.w3.org/ns/dcat#Distribution",
+    "dialect": {
+        "commentPrefix": null
+    },
+    "notes": [
+        {
+            "dc:title": "environment",
+            "properties": {
+                "glottolog_version": "",
```

Comment:
Interesting, this seems to be a Windows issues (it should be filled automatically - we can have a look at that).